### PR TITLE
Adding Machine Learning Working Group

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -1733,3 +1733,23 @@ workinggroups:
...
+    mission_statement: >
+      Identify and fix gaps in Kubernetes to better support Machine Learning applications
+    leads:
+    - name: Vishnu Kannan
+      github: vishh
+      company: Google
+    - name: Kenneth Owens
+      github: kow3ns
+      company: Google
+    meetings:
```
Comment (cblecker) [inline sigs.yaml]: This should look something like this:
```
     meetings:
     - description: Regular WG Meeting
       day: Thursday
       time: "13:00"
       tz: "PT (Pacific Time)"
       frequency: biweekly (On demand)
       url: https://zoom.us/j/4799874685
       archive_url: https://goo.gl/gBCdt1
```
and remove meeting_url and meeting_archive_url

Comment (vishh) [reply sigs.yaml]: Done.

**Comment by vishh** [comment]
@cblecker @kow3ns PTAL

**Pull request merged**